db/models/boycottLists.js

```diff
@@ -15,7 +14,7 @@ const BoycottList = db.define("boycottList", {
     allowNull: false,
   },
   reason: {
-    type: STRING,
+    type: TEXT,
```

Comment:
does text allow for a default length longer than string? i never used it before

Reply:
Yeah. A string is limited when it comes to length.

Reply:
Nice. good to know